GFW/convert.py

```diff
@@ -0,0 +1,170 @@
+import os
+import time
+import datetime
+import xarray as xr
+import rioxarray as rxr
+import numpy as np
+import cftime as cf
+import cfunits
+from urllib.request import urlretrieve
+
+# variables
+sdate = datetime.datetime(2020, 1, 1, 0, 0, 0)
+edate = datetime.datetime(2050, 1, 1, 0, 0, 0)
+proj = 'EPSG:4326'
+github_path = 'https://github.com/rubisco-sfa/ILAMB-Data/blob/msteckler/GFW/convert.py'
+
+# data sources
+remote_data = 'https://www.arcgis.com/sharing/rest/content/items/f950ea7878e143258a495daddea90cc0/data'
+local_data = 'sequestration_rate_mean_aboveground_full_extent_Mg_C_ha_yr.tif'
+remote_u_data = 'https://www.arcgis.com/sharing/rest/content/items/d28470313b8e443aa90d5cbcd0f74163/data'
+local_u_data = 'sequestration_error_ratio_layer_in_full_extent.tif'
+
+# function to open a raster and set the CRS
+def open_raster(local_data, remote_data):
+    if not os.path.isfile(local_data):
+        urlretrieve(remote_data, local_data)
+    download_stamp = time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(os.path.getmtime(local_data)))
+    data = rxr.open_rasterio(local_data, band_as_variable=True)
+    
+    # ensure data is not projected and is wgs84
+    epsg_code = int(data.rio.crs.to_epsg())
+    if epsg_code != 4326:
+        data = data.rio.reproject(crs=proj)
+
+    return data, download_stamp
+
+# open data and error files
+data, download_stamp = open_raster(local_data, remote_data)
+errd, _ = open_raster(local_u_data, remote_u_data)
+
+# function to resample to 0.5 degrees
+def coarsen(target_res, data):
+    resampled_data = data.coarsen(x=(int(target_res / abs(data.rio.resolution()[0]))),
+                                  y=(int(target_res / abs(data.rio.resolution()[1])))).mean()
+    return resampled_data
+
+# resample
+data = coarsen(0.5, data)
```

Comment:
General comment: xarray operations tend to work more efficiently if performed on datasets. You could have merged `data` and `errd` into a single dataset and then coarsened. *Don't change anything*, just something to think about.